domain-cc/cc-app/src/python_src/api.py

```diff
@@ -47,35 +50,51 @@ def get_health_status():
     return {"status": "ok"}
 
 
+def log_lookup_table_match(
+    classification_code: int, is_in_dropdown: bool, contention_text: str
+):
+    log_contention_text = contention_text if is_in_dropdown else "Not in dropdown"
+
+    if classification_code:
+        already_mapped_text = (  # being explicit, do not leak PII
+            contention_text.strip().lower()
+        )
+        log_as_json({"Lookup table match": already_mapped_text})
+    elif is_in_dropdown:
+        log_as_json({"No lookup table match for dropdown entry": log_contention_text})
+    else:
+        log_as_json("No Lookup table match for free text")
```

Comment:
This will always raise an error, since `log_as_json` can't assign `date` or `time` fields to a string.

Also, it may not be obvious to someone exploring the logs that they need to search for `"No lookup table match for dropdown entry" + "No Lookup table match for free text"` to return all the events where there wasn't a match. You might consider changing this to something like
```suggestion
        log_as_json("is_match": classification_code is not None, "is_in_dropdown": is_in_dropdown, "log_contention_text": log_contention_text)
```